tests/integration/run_tests.py

```diff
@@ -0,0 +1,215 @@
+            files = tar.getmembers()
+            if len(files) != 1:
+                print ("bad archive file " + json_filename)
+                sys.exit(1)
+            file = tar.extractfile(files[0])
+            buff = file.read()
+            tar.close()
+            jsonrpc_commands = json.loads(buff)
+    else:
+        with open(json_filename, encoding='utf8') as json_file_ptr:
+            jsonrpc_commands = json.load(json_file_ptr)
+    for json_rpc in jsonrpc_commands:
+        request = json_rpc["request"]
+        request_dumps = json.dumps(request)
+        target = get_target(silk, request["method"])
+        if verbose:
+            print (str(test_number) + ") " + request_dumps)
+        if verify_with_rpc == 0:
+            cmd = '''curl --silent -X POST -H "Content-Type: application/json" --data \'''' + request_dumps + '''\' ''' + target
+            cmd1 = ""
+            response = json_rpc["response"]
+            output_dir_name = ""
+            silk_file = ""
+            rpc_file = ""
+            diff_file = ""
+        else:
+            output_api_filename = output_dir + json_file[:-5]
+            output_dir_name = output_api_filename[:output_api_filename.rfind("/")]
+            response = ""
+            target = get_target(1, request["method"])
+            cmd = '''curl --silent -X POST -H "Content-Type: application/json" --data \'''' + request_dumps + '''\' ''' + target
+            target1 = get_target(0, request["method"])
+            cmd1 = '''curl --silent -X POST -H "Content-Type: application/json" --data \'''' + request_dumps + '''\' ''' + target1
+            silk_file = output_api_filename + "-silk.json"
+            rpc_file = output_api_filename + "-rpcdaemon.json"
+            diff_file = output_api_filename + "-diff.json"
+            run_shell_command(
+                cmd,
+                cmd1,
+                response,
+                verbose,
+                exit_on_fail,
+                output_dir_name,
+                silk_file,
+                rpc_file,
+                diff_file)
+
+#
+# usage
+#
+def usage(argv):
+    """ Print script usage
+    """
+    print("Usage: " + argv[0] + " -h -c -r -v -a <api_name> -t < test_number> -l < no of loops> -d")
+    print("")
+    print("Launch an automated test sequence on Silkrpc or RPCDaemon")
+    print("")
+    print("-h print this help")
+    print("-c runs all tests even if one test fails [ default exit at first test fail ]")
+    print("-r connect to rpcdaemon [ default connect to silk ] ")
+    print("-l <number of loops>")
+    print("-a <test api >")
+    print("-t <test_number>")
+    print("-d verify real time with rpc")
+    print("-v verbose")
+
+
+#
+# main
+#
+def main(argv):
+    """ parse command line and execute tests
+    """
+    exit_on_fail = 1
+    silk = 1
+    loop_number = 1
+    verbose = 0
+    req_test = -1
+    api_name = ""
+    verify_with_rpc = 0
+    json_dir = "./goerly/"
```

Comment:
Maybe the chain name could be a command-line option to support more than one chain. It's OK to do it in another PR